code/game/objects/items/weapons/grenades/smokebomb.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/grenade/smokebomb
 	name = "FS SG \"Reynolds\""
-	desc = "Smoke grenade, used to provide cover and hide movements."
+	desc = "Smoke grenade, used to create smoke cloud providing cover and hiding one's movement."
```

Comment:
try this instead

"Smoke grenade, used to create a cloud of smoke providing cover and hiding movement."